app/models/client/models/ChatMessage.js

```diff
@@ -91,6 +94,8 @@ const messagePreFetch = () => {
 		if (!messagesFetched && CachedChatSubscription.ready.get()) {
 			const status = Meteor.status();
 			if (status.status !== 'connected') {
+				clearTimeout(timeout);
```

Comment:
@Shailesh351 a small change here. Sometimes when you refresh, the status shows offline, and then online after some time, and `clearTimeout` is called twice. To avoid that timeout is used.